test/fixture/sessionEncryption.js

```diff
@@ -0,0 +1,25 @@
+const { JWK, JWE } = require('jose');
+const hkdf = require('futoin-hkdf');
+
+const deriveKey = (secret) => hkdf(secret, 32, { info: 'JWE CEK', hash: 'SHA-256' });
+const epoch = () => Date.now() / 1000 | 0;
```

Comment:
Is this working? Or the pipe should be an OR? Also, I'd add parentheses for clarifying the scope.

```suggestion
const epoch = () => (Date.now() / 1000) || 0;
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#.7c_%28Bitwise_OR%29

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#.7c_%28Bitwise_OR%29